server/dash/dashboards/neighborhood_recent.py

```diff
@@ -161,7 +149,26 @@ def populate_options():
     Input("council_list", "value")
 )
 def update_table(selected_council):
+    """Dash Callback Function to update the LA 311 request data table at the bottom of the dashboard.
+
+    This function takes the selected neighborhood council (nc) value from the "council_list" dropdown and 
+    outputs a list of requests associated with that nc as a data table in dictionary form 
+    with id "council_table" in the layout. 
+
+    Typical usage example:
+
+        dash_table.DataTable(
+            id='council_table',
+            ...
+        )
+    """
```

Comment:
Please add the `Returns` section. It would also be nice to see explicitly what is in the output dictionary (what are the keys, what are the values).

Reply:
Thanks for the suggestion - resolved.